src/aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -250,6 +250,12 @@ def detect_important_message(logs, line):
             'problems computing cholesky': 'ERROR_COMPUTING_CHOLESKY',
             'dexx is negative': 'ERROR_DEXX_IS_NEGATIVE',
             'too many bands are not converged': 'ERROR_DIAGONALIZATION_TOO_MANY_BANDS_NOT_CONVERGED',
+            'S matrix not positive definite': 'ERROR_S_MATRIX_NOT_POSITIVE_DEFINITE',
+            'zhegvd failed': 'ERROR_ZHEGVD_FAILED',
+            '[Q, R] = qr(X, 0) failed': 'ERROR_QR_FAILED',
+            'probably because G_par is NOT a reciprocal lattice vector': 'ERROR_G_PAR',
+            'eigenvectors failed to converge': 'ERROR_EIGENVECTOR_CONVERGENCE',
+            'factorization': 'ERROR_BROYDEN_FACTORIZATION',
```

Comment:
I am just wondering here if the regex would detect something else (e.g. a line containing this word). Wondering if there are lines having this word. From QE v7.2 this is eventually not a problem, as it is put nevertheless in the CRASH.

Reply:
Not 100% sure of this, but isn't the `Error in routine broyden (1):` error always the `factorization` one? :)

Reply:
Yes, so technically not a problem, correct?

Reply:
I mean you can look in the `stdout` for the `Error in routine broyden (1):` string. ^^ That's much less likely to appear elsewhere.